contract/start-airdrop-permit.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
any particular reason to check in a duplicate version of the permit?

Reply:
nope, I'll remove the files produced via the bundling process.